Extensions/dist/extensions.js

```diff
@@ -157,3 +167,11 @@ function writeListFile(extensions) {
   });
   writePageFile('list.json', JSON.stringify(list));
 }
+
+if (require.main === module) {
+	main();
+}
+
+module.exports = {
```

Comment:
This can actually just be `exports.build = build`.  It's also more in the NodeJS style to put this line near the function you're exporting (so after the function starting on line 16).